### PR TITLE
Camunda custom batch extension

### PR DESCRIPTION
**Pull request by pschalk**
Hi,

can you please have a look if the image in the blog post is shown correctly before publishing it? 
I was not able to start hugo at my machine :/

Best regards,
Patrick

### CODE REVIEW

**Comment by romansmirnov** [comment]
Hi @pschalk,

As far as I know @aakhmerov is your sponsor for that community extension. So I assigned him that pull request. Since he is not in the office this week, I think he will come back to you next week.

Cheers,
Roman

**Comment by pschalk** [comment]
Hi @romansmirnov,

thanks, he told me already that he will have a look when he is back.

Cheers,
Patrick

File: content/post/2017/06/camunda-bpm-custom-batch-1.0.0.md [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+If you have no clue what Camunda Batch is for, it could be used to split a huge workload (e.g. like reassigning all tasks) into small asynchronous jobs. 
+See official Camunda docs for more details: [Camunda Batch](https://docs.camunda.org/manual/7.6/user-guide/process-engine/batch/)
+
+<!--more-->
+
+# So How Does the Extension Work?
+
+The idea of the Camunda Batch is to separate the entire work called batch into smaller pieces called chunks
+First you have to define a job handler, which processes the batch chunks.
+All the boilerplate stuff like creating batch jobs,
```
Comment (aakhmerov) [inline content/post/2017/06/camunda-bpm-custom-batch-1.0.0.md]: may be rephrase? 

> All the boilerplate stuff like

is very slangy, I think. May be "work" is a better synonym for "stuff"?

File: content/post/2017/06/camunda-bpm-custom-batch-1.0.0.md [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+
+- Provide possibility for using of a data collector class
+- Provide a timer job for automatically triggering of batch creation
+- Provide a service bean for creation of the batch (additionally to the builder)
+- Provide helper utils to simplify testing
+- And many more... 
+
+# Last but Not Least?
+
+If you have any questions or ideas, feel free to contact me, or much more better, start contributing to the extension. :)
```
Comment (aakhmerov) [inline content/post/2017/06/camunda-bpm-custom-batch-1.0.0.md]: > or much more better

has to be rephrased, at least in "much better"

File: content/post/2017/06/camunda-bpm-custom-batch-1.0.0.md [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+
+There are a lot of ideas which are planned for the next versions:
+
+- Provide possibility for using of a data collector class
+- Provide a timer job for automatically triggering of batch creation
+- Provide a service bean for creation of the batch (additionally to the builder)
+- Provide helper utils to simplify testing
+- And many more... 
+
+# Last but Not Least?
```
Comment (aakhmerov) [inline content/post/2017/06/camunda-bpm-custom-batch-1.0.0.md]: Does question mark in the end make sense?

**Comment by mburke84** [comment]
Hi @pschalk,
Thanks for contributing this blog post. I've proofread it and it's live now :)

Best regards,
Mat

**Pull request merged**